pkg/apis/istio/v1alpha2/istiocontrolplane_types.pb.go

```diff
@@ -191,8 +190,6 @@ func (InstallStatus_Status) EnumDescriptor() ([]byte, []int) {
 
 // IstioControlPlane is a CustomResourceDefinition (CRD) describing an Istio control plane.
 type IstioControlPlane struct {
-	v11.ObjectMeta `json:"metadata,omitempty"`
```

Comment:
ObjectMeta and TypeMeta are required.  I am not sure why these are omiited in your PR.

Reply:
Oh, I missed that these were removed.

Arg, there's something funny happening with the code gen in this repo :-(